charts/seed-bootstrap/dashboards/vpa-recommender.json

```
@@ -0,0 +1,1197 @@
+      "type": "row"
+    }
+  ],
+  "refresh": "1m",
+  "schemaVersion": 27,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "UpdateVPAs",
+          "value": "UpdateVPAs"
+        },
+        "datasource": "seed-prometheus",
+        "definition": "label_values(step)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "step",
+        "multi": false,
+        "name": "step",
+        "options": [],
+        "query": {
+          "query": "label_values(step)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "shoot--garden--cp-mgr-azure",
+          "value": "shoot--garden--cp-mgr-azure"
+        },
+        "datasource": "seed-prometheus",
+        "definition": "label_values(vpa_recommender_vpa_objects_count,namespace)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "namespace",
+        "options": [],
+        "query": {
+          "query": "label_values(vpa_recommender_vpa_objects_count,namespace)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": ".+",
+        "current": {
+          "selected": false,
+          "text": "vpa-recommender-7cfc84668b-2wc2f",
+          "value": "vpa-recommender-7cfc84668b-2wc2f"
+        },
+        "datasource": "seed-prometheus",
+        "definition": "label_values(vpa_recommender_vpa_objects_count{namespace=\"$namespace\"},pod)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "vpa",
+        "options": [],
+        "query": {
+          "query": "label_values(vpa_recommender_vpa_objects_count{namespace=\"$namespace\"},pod)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-15m",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
```

With https://github.com/gardener/gardener/pull/4229/files we made all dashboard to use UTC by default. Can this dashboard set the timezone explicitly to prevent confusion on which timezone is used?

```suggestion
  "timezone": "utc",
```

---

Good catch! The browser displayed 'UTC', so I didn't even bother to understand more about the dashboard settings for timezones!